include/deal.II/lac/trilinos_sparse_matrix.h

```diff
@@ -1475,15 +1475,14 @@ namespace TrilinosWrappers
      * Source and destination must not be the same vector.
      *
      * This function can be called with several different vector objects,
-     * namely TrilinosWrappers::VectorBase, TrilinosWrappers::MPI::Vector as well
-     * as deal.II's own vector classes Vector<double> and
-     * LinearAlgebra::distributed::Vector<double>.
+     * namely TrilinosWrappers::MPI::Vector as well as deal.II's own vector
+     * classes Vector<double> and LinearAlgebra::distributed::Vector<double>.
```

Comment:
I think we are missing an `e.g.,` before `Vector<double>` since we don't list all of our own vector classes.